app/views/promotion/newspaperLandingPage.scala.html

```diff
@@ -0,0 +1,92 @@
+@import com.gu.memsub.promo.PromoCode
+@import com.gu.memsub.promo.Promotion.PromoWithNewspaperLandingPage
+@import com.gu.memsub.promo.Promotion.asAnyPromotion
+@import views.support.LandingPageOps._
+@import views.support.MarkdownRenderer
+@import com.gu.memsub.subsv2.Catalog
+@import model.Subscriptions._
+
+@(catalog: Catalog, promoCode: PromoCode, promotion: PromoWithNewspaperLandingPage, md: MarkdownRenderer)
+@voucherPlans = @{catalog.voucher.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
+@deliveryPlans = @{catalog.delivery.list.filter(p => promotion.appliesTo.productRatePlanIds.contains(p.id))}
+@anyPromotion = @{asAnyPromotion(promotion)}
+@title = @{"Subscribe to the Guardian and Observer"}
+@hash = @{s"#${promotion.landingPage.defaultProduct}"}
+@products = @{
```

Comment:
that's a lot of code to inline into a template, I'd be very tempted to pull it out into an object so it can compile separately